src/pages/workspaces/[workspaceSlug]/pipelines/[pipelineCode]/runs/[runId].tsx

```diff
@@ -84,6 +84,8 @@ const WorkspacePipelineRunPage: NextPageWithLayout = (props: Props) => {
     PipelineRunStatus.Success,
   ].includes(run.status);
 
+  const runOuputs = [...run.datasetVersions, run.outputs];
```

Comment:
It's not ideal to build a list to just check if there is an element in it. I'd do a simple

`hasOutputs = (run.datasetVersions.length + run.outputs.length) > 0